packages/core/core/src/services/server/register-routes.ts

```diff
@@ -82,8 +82,8 @@ const registerPluginRoutes = (strapi: Core.Strapi) => {
  * Register api routes
  */
 const registerAPIRoutes = (strapi: Core.Strapi) => {
-  for (const apiName of Object.keys(strapi.api)) {
-    const api = strapi.api[apiName];
+  for (const apiName of Object.keys(strapi.apis)) {
```

Comment:
Just a random thought, but it could be nice to implement some `[key, item][]` array methods to the modules, so that you could do

```ts
strapi.apis.forEach([apiName, api]) => {...})
strapi.apis.map([apiName, api]) => {...})
strapi.apis.filter([apiName, api]) => {...})
```

Reply:
this is strapi.get('apis') basically. the shortcut just returns an object from the map. I actually would like to just return the registry. just did this for consistency and I can change all the shortcut at once afterwards 👍